tests/python/unittest/test_higher_order_grad.py

```diff
@@ -106,6 +106,35 @@ def grad_grad_op(x):
         check_second_order_unary(array, log10, grad_grad_op)
 
 
+@with_seed()
+def test_reciprocal():
+    def reciprocal(x):
+        return nd.reciprocal(x)
+
+    def grad_grad_op(x):
+        return 2/x**3
+
+    for dim in range(1, 5):
+        shape = rand_shape_nd(dim)
+        array = random_arrays(shape)
+        check_second_order_unary(array, reciprocal, grad_grad_op)
+
+
+@with_seed()
+def test_abs():
+    def abs(x):
+        return nd.abs(x)
+
+    def grad_grad_op(x):
+        return nd.zeros_like(x)
+
+    for dim in range(1, 5):
+        shape = rand_shape_nd(dim)
+        array = random_arrays(shape)
+        check_second_order_unary(array, abs, grad_grad_op)
+
```

Comment:
nit: please remove extra line

Reply:
two lines between functions as per pep8

Reply:
It is fixed actually. I guess I removed the lower line so it is not showing up here.